app/replica.go

```diff
@@ -190,6 +214,22 @@ func startReplica(c *cli.Context) error {
 		}
 		go AutoConfigureReplica(s, frontendIP, "tcp://"+address, replicaType)
 	}
+	for s.Replica() == nil {
+		time.Sleep(2 * time.Second)
+	}
+	if replicaType == "clone" && snapName != "" {
+		logrus.Infof("Starting clone process\n")
+		status := s.Replica().GetCloneStatus()
+		if status != "completed" {
+			if err = CloneReplica(s, "tcp://"+address, cloneIP, snapName); err != nil {
+				return err
+			}
+		}
+		s.Replica().SetCloneStatus("completed")
```

Comment:
What type of statuses do we have?  just completed and NA? what about in progress? error? etc?

Reply:
So why not add state that's in progress? If someone else needs to find out he/she would need to find this PR to understand the reasoning.